ui/src/layouts/storage.rs

```diff
@@ -844,3 +858,9 @@ async fn drag_and_drop_function(
         tokio::time::sleep(std::time::Duration::from_millis(100)).await;
     }
 }
+
+// Decodes the given path from an html encoded path
+#[cfg(target_os = "linux")]
+pub fn decoded_path_string(path: &Path) -> String {
```

Comment:
Maybe it would be best to return a `PathBuf` here instead.

Reply:
now gets passed the vec<pathbuf> and returns that